subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/tasks/InstallXCTestBundle.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.nativeplatform.test.xctest.tasks;
+
+import org.apache.commons.io.FilenameUtils;
+import org.gradle.api.Action;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Incubating;
+import org.gradle.api.Transformer;
+import org.gradle.api.file.CopySpec;
+import org.gradle.api.file.Directory;
+import org.gradle.api.file.DirectoryProperty;
+import org.gradle.api.internal.file.FileOperations;
+import org.gradle.api.tasks.InputDirectory;
+import org.gradle.api.tasks.Internal;
+import org.gradle.api.tasks.OutputDirectory;
+import org.gradle.api.tasks.SkipWhenEmpty;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.internal.nativeintegration.filesystem.FileSystem;
+import org.gradle.util.GFileUtils;
+
+import javax.inject.Inject;
+import java.io.File;
+
+/**
+ * Installs a XCTest bundle with a run script so it can be easily executed.
+ *
+ * @since 4.4
+ */
+@Incubating
+public class InstallXCTestBundle extends DefaultTask {
+    private final DirectoryProperty bundleDirectory = newInputDirectory();
+    private final DirectoryProperty installDirectory = newOutputDirectory();
+
+    @Inject
+    protected FileSystem getFileSystem() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Inject
+    protected FileOperations getFileOperations() {
+        throw new UnsupportedOperationException();
+    }
+
+    @TaskAction
+    void install() {
```

Comment:
I'm not sure if this suggestion makes sense... but couldn't we also install the bundle directly into the final directory?  IOW, we could rename `CreateSwiftBundle` to `InstallXCTestBundle` and do everything in there. I think we would have to do a little more moving around of things so that creating a SwiftBundle wasn't a generic feature and something that's just specific to running XCTests on macOS.

Reply:
Done.